crates/trycp_server/src/main.rs

```diff
@@ -62,6 +63,12 @@ async fn main() -> Result<(), Error> {
             default_value = "9000"
         )]
         port: u16,
+        #[structopt(
+            long = "lair-shim",
+            short = "ls",
```

Comment:
I would rather remove this shorthand. It's not obvious what it stands for to someone who's familiar with trycp-server.
```suggestion
```